vue-components/src/components/InputWithExtender.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+	<div
+		:class="[ 'wikit', 'wikit-InputWithExtender' ]"
+	>
+		<label class="wikit-InputWithExtender__label" :for="id">{{ label }}</label>
+		<Input
+			:id="id"
+			:value="value"
+			@focus.native="showExtension = true"
+			@blur.native="showExtension = false"
+			@esc.native="showExtension = false"
+			@input="onInput"
+			:placeholder="placeholder"
+			:disabled="disabled"
+		/>
```

Comment:
This is missing the feedbackType prop, so the border around the input with an error message are currently not red. See the TextInput for an example.

Reply:
Done